### ISSUE TITLE
Data Dictionary doesn't work for Query CSVs

### ISSUE THREAD

**Issue by jqnatividad** [ckan/ckan#4148]
### CKAN Version if known (or site URL)
This is for https://github.com/ckan/ckan/tree/3816-query-based-views

### Please describe the expected behaviour
For a Query CSV, be able to add Data Dictionary items.

### Please describe the actual behaviour
One gets a server error, looking at the error log, the system is trying to manipulate the "_full_text" column and create an index on it.

**Comment by wardi** [comment]
good catch. I was testing the dictionaries with a query that included a `select *` from another table so I missed this one. Will take a look

**Comment by jqnatividad** [comment]
A related issue, but thought to just add it to this issue instead of creating another one - Last Updated is not updated when the Query is modified.
